packages/asthma_diary_app/lib/feature/peak_flow/ui/component/peak_flow_chart.dart

```diff
@@ -178,28 +180,85 @@ class CurrentPeakFlowChartRangeText extends ConsumerWidget {
 
 /// 表示期間のセレクタ
 class _ChartRangeSelector extends ConsumerWidget {
-  const _ChartRangeSelector();
+  const _ChartRangeSelector({
+    required this.pageController,
+  });
+
+  final PageController pageController;
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final current = ref.watch(currentChartPeriodTypeNotifierProvider);
-    return Row(
-      mainAxisAlignment: MainAxisAlignment.center,
+    return Stack(
       children: [
-        for (final ChartPeriodType period in ChartPeriodType.values) ...{
-          _ChartRangeSelectorButton(
-            item: period,
-            selected: period == current,
-            onPressed: () => ref
-                .watch(currentChartPeriodTypeNotifierProvider.notifier)
-                .updatePeriod(period),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            for (final ChartPeriodType period in ChartPeriodType.values) ...{
+              _ChartRangeSelectorButton(
+                item: period,
+                selected: period == current,
+                onPressed: () => ref
+                    .watch(currentChartPeriodTypeNotifierProvider.notifier)
+                    .updatePeriod(period),
+              ),
+            },
+          ],
+        ),
+        Positioned(
+          right: _horizontalPadding,
+          child: _MoveToNowButton(
+            pageController: pageController,
           ),
-        },
+        ),
       ],
     );
   }
 }
 
+/// 現在に移動するボタン
+class _MoveToNowButton extends ConsumerWidget {
+  const _MoveToNowButton({
+    required this.pageController,
+  });
+
+  final PageController pageController;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final currentType = ref.watch(currentChartPeriodTypeNotifierProvider);
+    final currentRange =
+        ref.watch(currentPeakFlowChartRangeProvider).valueOrNull;
+
+    return Visibility(
+      visible: currentRange != null && !currentRange.contains(DateTime.now()),
+      child: TextButton(
+        onPressed: () => _moveToNow(ref),
+        style: TextButton.styleFrom(
+          foregroundColor: context.onSecondary,
+          backgroundColor: context.secondary,
+        ),
+        child: Text(currentType.currentLabel),
+      ),
```

Comment:
他のボタンと同化しているので、次のようなアイコンボタンにしてほしいです！

```suggestion
     child: IconButton(
        onPressed: () => _moveToNow(ref),
        icon: const Icon(Icons.today),
      ),
```